src/components/Main/Results/DeterministicLinePlot.tsx

```diff
@@ -230,7 +239,11 @@ export function DeterministicLinePlot({ data, userResult, logScale, showHumanize
                   tickCount={7}
                 />
                 <YAxis scale={logScaleString} type="number" domain={[1, 'dataMax']} tickFormatter={yTickFormatter} />
-                <Tooltip formatter={tooltipFormatter} labelFormatter={labelFormatter} />
+                <Tooltip 
+                  formatter={tooltipFormatter} 
+                  position={tooltipPosition} 
```

Comment:
Can you pass `labelFormatter={labelFormatter}` also so that `<ResponsiveTooltipContent />` does not have to guess as to label type i.e. date -v- age.